index.js

```diff
@@ -17,7 +17,8 @@
 'use strict';
 
 var OpaqueSpan = require('./lib/opaque-span.js');
-var Logger = require('./lib/logger.js');
+var CloudCommon = require('@google/cloud-diagnostics-common');
+var Logger = CloudCommon.logger;
```

Comment:
ditto. This should be `var Logger = common.Logger;` as the latter is actually a class, so it should exported from that module as such.